### PR TITLE
BootstrapStyle-Changes PR#4

### PR DESCRIPTION
**Pull request by NihilMaster**
This is a Pull Request I created to review the changes I made when applying Bootstrap Style.

### CODE REVIEW

File: app/frontend/stylesheets/_bootswatch.scss [hunk 1]
```diff
@@ -0,0 +1,124 @@
+// Darkly 5.0.1
```
Comment (mauriciosabogal) [inline app/frontend/stylesheets/_bootswatch.scss:1]: What's the difference between this file and `app/assets/stylesheets/_bootswatch.scss`?
Why do you need both?

File: app/assets/stylesheets/bootstrap.min.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*!
```
Comment (mauriciosabogal) [inline app/assets/stylesheets/bootstrap.min.css]: Why is this file required? You already installed Bootstrap.

File: app/assets/stylesheets/bootstrap.css [hunk 3]
```diff
@@ -0,0 +1,11031 @@
+/*!
```
Comment (mauriciosabogal) [inline app/assets/stylesheets/bootstrap.css:1]: Why is this file required? You already installed Bootstrap.

File: app/assets/css/stylesold.css [hunk 4]
```diff
@@ -0,0 +1,10835 @@
+@charset "UTF-8";
```
Comment (mauriciosabogal) [inline app/assets/css/stylesold.css]: What is the purpose of this file?

File: Gemfile.lock [hunk 5]
```diff
@@ -62,9 +62,15 @@ GEM
       zeitwerk (~> 2.3)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
+    autoprefixer-rails (10.2.5.1)
+      execjs (> 0)
     bindex (0.8.1)
     bootsnap (1.7.5)
       msgpack (~> 1.0)
+    bootstrap (5.0.1)
```
Comment (mauriciosabogal) [inline Gemfile.lock]: Is this gem required? You already added Bootstrap to `package.json`.

File: app/frontend/stylesheets/styles.scss [hunk 6]
```diff
@@ -0,0 +1,173 @@
+// Darkly 5.0.1
```
Comment (mauriciosabogal) [inline app/frontend/stylesheets/styles.scss]: What's the difference between `app/frontend/stylesheets/styles.scss` and `app/assets/css/styles.scss` ?

Comment (NihilMaster) [reply app/frontend/stylesheets/styles.scss]: Duplicate file. The functional file is `app/assets/css/styles.scss`
**Fixed**

File: Gemfile.lock (hunk 5 above)
Comment (NihilMaster) [reply Gemfile.lock]: No, that’s why it’s not in the Gemfile either. In my local repository, it is no longer added to the Gemfile.lock since I ran a Bundle Install after committing.

File: app/assets/css/stylesold.css (hunk 4 above)
Comment (NihilMaster) [reply app/assets/css/stylesold.css]: It was an old version of a file I forgot to remove from the project.

File: app/assets/stylesheets/bootstrap.min.css (hunk 2 above)
Comment (NihilMaster) [reply app/assets/stylesheets/bootstrap.min.css]: Unnecessary file. Deleted.

File: app/frontend/stylesheets/_bootswatch.scss (hunk 1 above)
Comment (NihilMaster) [reply app/frontend/stylesheets/_bootswatch.scss:1]: I don’t need both. The functional file is `app/frontend/stylesheets/_bootswatch.scss`.
**Fixed**

File: app/assets/stylesheets/bootstrap.css (hunk 3 above)
Comment (NihilMaster) [reply app/assets/stylesheets/bootstrap.css:1]: This file was the only way I got the template to load. Even though without it, I still have Bootstrap, but template style isn't loaded.

**Review by mauriciosabogal — APPROVED** [review]
